src/merge.py

```
@@ -579,19 +542,45 @@ def main():
         filtered_icloud_events = [event for event in icloud_events if event.title == source_tag]
+        merge_events:list[MergeEvent] = []
+        event_addition = False
         term.print_done()
 
-        print_step(TAG_CALENDAR_MERGE, "reconciling events...", one_liner=False)
-        merge_events, event_addition = _reconcile_events(filtered_icloud_events, source_calendar_events)
-        if event_addition:
-            for event in merge_events:
-                if event.action == EventAction.add:
-                    event.title = source_tag
+        print_step(TAG_CALENDAR_MERGE, f"identifying events to remove...", False)
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)